Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -598,7 +598,7 @@ WarpXParticleContainer::DepositCharge (amrex::Vector<std::unique_ptr<amrex::Mult
         if (reset) rho[lev]->setVal(0.0, rho[lev]->nGrow());
 
         // Loop over particle tiles and deposit charge on each level
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
 #pragma omp parallel
```

Comment:
Missing `if (amrex::Gpu::notInLaunchRegion())`?

Reply:
I agree with you. In principle `if (amrex::Gpu::notInLaunchRegion())` should be there

Reply:
```suggestion
#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
```